sparkle/src/icons/index.ts

```diff
@@ -0,0 +1,234 @@
+  CheckCircle as CheckCircleIcon,
+  Check as CheckIcon,
+  ChevronDown as ChevronDownIcon,
+  ChevronLeft as ChevronLeftIcon,
+  ChevronRight as ChevronRightIcon,
+  ChevronUpDown as ChevronUpDownIcon,
+  ChevronUp as ChevronUpIcon,
+  Circle as CircleIcon,
+  ClipboardCheck as ClipboardCheckIcon,
+  Clipboard as ClipboardIcon,
+  Clock as ClockIcon,
+  CloudArrowDown as CloudArrowDownIcon,
+  CloudArrowLeftRight as CloudArrowLeftRightIcon,
+  CloudArrowUp as CloudArrowUpIcon,
+  CloudX as CloudXIcon,
+  Cog6Tooth as Cog6ToothIcon,
+  Command as CommandIcon,
+  CommandLine as CommandLineIcon,
+  Cube as CubeIcon,
+  Dash as DashIcon,
+  DocumentDuplicate as DocumentDuplicateIcon,
+  Document as DocumentIcon,
+  DocumentPile as DocumentPileIcon,
+  DocumentPlus as DocumentPlusIcon,
+  DocumentText as DocumentTextIcon,
+  Dust as DustIcon,
+  EmotionLaugh as EmotionLaughIcon,
+  Equalizer as EqualizerIcon,
+  ExclamationCircle as ExclamationCircleIcon,
+  ExternalLink as ExternalLinkIcon,
+  Eye as EyeIcon,
+  EyeSlash as EyeSlashIcon,
+  Flag as FlagIcon,
+  Folder as FolderIcon,
+  FolderOpen as FolderOpenIcon,
+  FullscreenExit as FullscreenExitIcon,
+  Fullscreen as FullscreenIcon,
+  GlobeAlt as GlobeAltIcon,
+  Grab as GrabIcon,
+  HandHeart as HandHeartIcon,
+  HandThumbDown as HandThumbDownIcon,
+  HandThumbUp as HandThumbUpIcon,
+  HeartAlt as HeartAltIcon,
+  Hexagon as HexagonIcon,
+  Image as ImageIcon,
+  InformationCircle as InformationCircleIcon,
+  Key as KeyIcon,
+  Lightbulb as LightbulbIcon,
+  Link as LinkIcon,
+  ListAdd as ListAddIcon,
+  ListCheck as ListCheckIcon,
+  List as ListIcon,
+  ListRemove as ListRemoveIcon,
+  Lock as LockIcon,
+  Login as LoginIcon,
+  Logout as LogoutIcon,
+  Magic as MagicIcon,
+  MagnifyingGlass as MagnifyingGlassIcon,
+  Menu as MenuIcon,
+  More as MoreIcon,
+  MovingMail as MovingMailIcon,
+  Paint as PaintIcon,
+  PaperAirplane as PaperAirplaneIcon,
+  Pause as PauseIcon,
+  PencilSquare as PencilSquareIcon,
+  Planet as PlanetIcon,
+  Play as PlayIcon,
+  PlusCircle as PlusCircleIcon,
+  Plus as PlusIcon,
+  Pushpin as PushpinIcon,
+  Puzzle as PuzzleIcon,
+  QuestionMarkCircle as QuestionMarkCircleIcon,
+  Reaction as ReactionIcon,
+  Rectangle as RectangleIcon,
+  Robot as RobotIcon,
+  Rocket as RocketIcon,
+  Scan as ScanIcon,
+  Server as ServerIcon,
+  ShakeHands as ShakeHandsIcon,
+  Shapes as ShapesIcon,
+  Sparkles as SparklesIcon,
+  Square3Stack3D as Square3Stack3DIcon,
+  Square as SquareIcon,
+  Star as StarIcon,
+  Stop as StopIcon,
+  Table as TableIcon,
+  Template as TemplateIcon,
+  TestTube as TestTubeIcon,
+  Time as TimeIcon,
+  Trash as TrashIcon,
+  Triangle as TriangleIcon,
+  UserArrow as UserArrowIcon,
+  UserGroup as UserGroupIcon,
+  User as UserIcon,
+  UserPlus as UserPlusIcon,
+  Wrench as WrenchIcon,
+  XCircle as XCircleIcon,
+  XMark as XMarkIcon,
+} from "./solid";
+export {
+  ArrowDownCircle as ArrowDownCircleStrokeIcon,
```

Comment:
So we export the same icons into different names?

Reply:
not in main export, but internally yes. I'll check if these names are used, I don't see the point of keeping both names